man/docker-run.1.md

```diff
@@ -190,8 +191,13 @@ the other shell to view a list of the running containers. You can reattach to a
 detached container with **docker attach**. If you choose to run a container in
 the detached mode, then you cannot use the **-rm** option.
 
-   When attached in the tty mode, you can detach from a running container without
-stopping the process by pressing the keys CTRL-P CTRL-Q.
+   When attached in the tty mode, you can detach from the container (and leave it
+running) using a configurable key sequence. The default sequence is `CTRL-p CTRL-q`.
+You configure the key sequence using the **--detach-keys** option or a configuration file.
+See **docker(1)** for documentation on using a configuration file.
+
+**--detach-keys**=""
+   Define the key sequence to use to detach from the container.
```

Comment:
`Override the key sequence for detaching a container. Format is ctrl-<value> where <value> is one of: a-z, @, ^, [, ``, or _`